setup.py

```diff
@@ -2,19 +2,42 @@
 
 from setuptools import setup
 
+import i18n
```

Comment:
I see the value in being able to programmatically determine the package version when debugging. I also see @nedbat's point. However, if the package fails to install, the tests won't run; so, we do have protection against a developer breaking the package.